WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -464,6 +464,25 @@ class GutenbergViewController: UIViewController, PostEditor {
         gutenberg.setFocusOnTitle()
     }
 
+    private func handleMissingBlockAlertButtonPressed() {
+        let blog = post.blog
+        let JetpackSSOEnabled = (blog.jetpack?.isConnected ?? false) && (blog.settings?.jetpackSSOEnabled ?? false)
+        if JetpackSSOEnabled == false {
+            let controller = JetpackSecuritySettingsViewController(blog: blog)
+            controller.navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(jetpackSettingsControllerDismissed))
+            let navController = UINavigationController(rootViewController: controller)
+            present(navController, animated: true)
+        }
+    }
+
+    @objc private func jetpackSettingsControllerDismissed() {
```

Comment:
Similar to `handleMissingBlockAlertButtonPressed`, would this be better named `jetpackSettingsControllerDoneButtonPressed` or similiar?